sample/android/src/main/java/com/trikot/viewModels/sample/databinding/ListItemViewModelAdapter.kt

```diff
@@ -6,14 +6,7 @@ import com.mirego.trikot.viewmodels.adapter.GenericViewModelDiffCallback
 import com.mirego.trikot.viewmodels.adapter.ViewModelAdapter
 import com.trikot.viewmodels.sample.BR
 import com.trikot.viewmodels.sample.R
-import com.trikot.viewmodels.sample.viewmodels.ButtonListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.HeaderListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.ImageListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.InputTextListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.LabelListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.NavigableListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.SliderListItemViewModel
-import com.trikot.viewmodels.sample.viewmodels.ViewListItemViewModel
+import com.trikot.viewmodels.sample.viewmodels.*
```

Comment:
Tes settings d'import ne sont pas bon, il ne faut pas mettre d'import `.*`

Reply:
ca le fait tout seul ... je vais fix ca

Reply:
You still have a `viewmodels.*` on autoformat even when using the "Use single name import" Kotlin Code Style settings?
![Screen Shot 2021-02-02 at 13 09 43](https://user-images.githubusercontent.com/5982196/106643581-ebf20900-6557-11eb-94e3-6f79935f6cf1.png)

Reply:
Normalement tu devrais pouvoir empêcher un wildcard dans la config Editor > Code Style > Kotlin > Imports